src/Altinn.Platform/Altinn.Platform.Events/Events/appsettings.json

```diff
@@ -19,7 +19,8 @@
   "PlatformSettings": {
     "ApiRegisterEndpoint": "http://localhost:5101/register/api/v1/",
     "ApiProfileEndpoint": "http://localhost:5101/profile/api/v1/",
-    "ApiAuthorizationEndpoint": "http://localhost:5101/authorization/api/v1/"
+    "ApiAuthorizationEndpoint": "http://localhost:5101/authorization/api/v1/",
+    "AppsDomain":  "apps.altinn.no"
```

Comment:
Would add any value to use altinn3local as domain here w.r.t. testing locally during development?

Reply:
hmm. Never tought about that as a possibility. Is that something that we have done for other components? Used local platform as platform for other platform components?

Reply:
I use that when testing storage, but might only be that authorization and those components are what we need. Let's leave it as is for now.